### DISCUSSION TITLE
Usar LPD como token de fidelización

### DISCUSSION THREAD

**Discussion by IBaixauli** [NGDAOorg/lapalma#2] [Ideas]
Propongo usar LPD como token de fidelización en comercios.
Los comercios pueden hacer descuentos a sus clientes en base a los LPDs entregados por el cliente
El comercio establece:

- % máximo de descuento aplicable a cada compra
- Precio al que valora cada LPD
- Fecha máxima de aceptación de cada LPD

El comerciante puede variar estos datos cuando estime oportuno
El listado y las condiciones de cada comercio se publicarán en la web de La Palma DAO

**Comment by Miller1985** [comment]
¿Cómo consigo LPDs?
1- Entrar en https://sushi.com/
2- Arriba a la derecha pone: Entrar en app. hacer click allí.
3- Conectar nuestra billetera de Metamask en la cual tendremos tokens para intercambiar por LPD (Matic, Ethereum, Dai...)
4- En la pantalla que nos aparece, saldrá un rectángulo. En el que en la parte superior pondremos el token que intercambiaremos por futuros LPD.
5- En la parte inferior tendremos que poner la dirección del contrato inteligente del LPD.
